docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,736 @@
+---
+title: "Build a Flutter App that Integrates with Viam"
+linkTitle: "Build a Flutter App"
+type: "docs"
+description: "Use Viam's Flutter SDK to build a custom mobile app to show your machines and their components."
+webmSrc: "/tutorials/flutter-app/demo.webm"
+mp4Src: "/tutorials/flutter-app/demo.mp4"
+videoAlt: "An example Viam-integrated Flutter app."
+tags: ["sdk", "flutter"]
+authors: ["Clint Purser"]
+languages: ["flutter"]
+viamresources: ["base"]
+level: "Intermediate"
+date: "2024-01-17"
+cost: "0"
+---
+
+<div class="td-max-width-on-larger-screens">
+ <div class="alignleft" style="max-width:250px; margin-right:40px">
+  {{<gif webm_src="/tutorials/flutter-app/demo.webm" mp4_src="/tutorials/flutter-app/demo.mp4" alt="Rendering of the mobile app. Log in is clicked, then all Clint's locations are shown, Clint's Desk it clicked, all its smart machines are listed, and then desk-robot is clicked, showing a list of components and services belonging to that smart machine.">}}
+  </div>
+</div>
+
+Flutter is Google's user interface toolkit for building applications for mobile, web, and desktop from a single codebase.
+If the existing [Viam mobile app](/fleet/#the-viam-mobile-app) doesn't fit your needs, you can use Viam's [Flutter SDK](https://flutter.viam.dev/) to build a custom app to interact with your smart machines that run on Viam.
```

Comment:
Okay yeah! Just adding link and fixing captial
```suggestion
If you're looking to monitor and control individual machines with the same functionality you have on the [**Control** tab](/fleet/machines/#control), you can use the general purpose [Viam mobile app](/fleet/#the-viam-mobile-app) rather than creating your own.
	If you need custom functionality or a custom interface, you can use Viam's [Flutter SDK](https://flutter.viam.dev/) to build a custom app to interact with your machines that run on Viam.
```